state/runtime/precompiled/base_test.go

```diff
@@ -13,6 +14,20 @@ type precompiledTest struct {
 	Name     string
 	Input    string
 	Expected string
+	Gas      uint64
+}
+
+var enabledForks = &chain.ForksInTime{
```

Comment:
I think you could just use `AllForksEnabled.At(0)`where needed without global variable